frappe/utils/background_jobs.py

```diff
@@ -13,15 +13,19 @@
 import pymysql
 from pymysql.constants import ER
 
-default_timeout = 300
-queue_timeout = {
-	'background': 2500,
-	'long': 1500,
-	'default': 300,
-	'short': 300
-}
-
 redis_connection = None
+queue_timeout  = {}
+
+def get_queues_and_timeout():
+	site_config = frappe.get_site_config(sites_path=".")
+	global queue_timeout
+
+	if queue_timeout:
+		return queue_timeout
+
+	for key, value in site_config["queues"].items():
```

Comment:
@sunnyakaxd : There is a path written in https://github.com/frappe/bench/pull/739 to handle.
The default queues will be set during update and further on new instalations will have these by default in common_site_config

Reply:
Still they would break in python 2 as there dictionary items are accessed via dict.iteritems() instead of dict.items()
six.iteritems(dict) takes care of compatibility.

Reply:
No. It will not. .items() is functional in both python2 and python3.

Reply:
Ah yep. My bad :P It'll work